src/include/optimizer/rules/unnesting_rules.h

```diff
@@ -49,7 +49,7 @@ class UnnestMarkJoinToInnerJoin : public Rule {
 };
 
 /**
- * Unnest Single Join to Inner Join
+ * Transform Single Join to Inner Join
```

Comment:
Since there's a file called transformation_rules, I think Unnest should still be the term here.